### PR TITLE
update dependecies

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

**Comment by norkans7** [comment]
requires https://github.com/nyaruka/smartmin/pull/45/

File: setup.py [hunk 1]
```diff
@@ -4,8 +4,8 @@
     name = 'dash',
     version = '0.0.1',
     licence = 'BSD',
-    install_requires = ['django==1.7',
-                        'smartmin==1.7',
+    install_requires = ['django==1.7.1',
+                        'smartmin==1.7.0',
```
Comment (nicpottier) [inline setup.py]: I don't think we want smartmin 1.7.0 here, instead we just want the dependency link below.

Comment (norkans7) [inline setup.py]: Right, I removed the version

File: pip-requires.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
--e git+https://github.com/nyaruka/smartmin.git@to_django_1_7#egg=smartmin-dev
--e git+https://github.com/django/django.git@stable/1.7.x#egg=django
+-e git+https://github.com/nyaruka/smartmin.git#egg=smartmin-dev
+Django==1.7.1
```
Comment (rowanseymour) [inline pip-requires.txt]: Any reason to not use 1.7.2 ?

Comment (nicpottier) [inline pip-requires.txt]: Agree, let's do 1.7.2

Comment (norkans7) [inline pip-requires.txt]: Done

**Pull request merged**